pravega_provisioner/cluster_provisioner.py

```diff
@@ -19,11 +20,16 @@ def get_bool_input(msg):
     :param msg: Message to be prompted to the user.
     :return: Valid user input.
     """
-    user_input = input(msg + " (yes, no)")
-    while user_input != "yes" and user_input != "no":
-        user_input = input(msg + " (yes, no)")
-    return user_input == "yes"
-
+    return_value = False
```

Comment:
done. yes it worked with python3.

Reply:
Reverted dir name change from pravega-provisioner to pravega_provisioner.
I figured our a way to pravega-provisioner as module,  with dash in name python is complaining, while downloading sources to scripts distribution I am copying to directory pravega_provisioner, so that the scale script which is calling into provisioner work.